src/notify.js

```diff
@@ -59,6 +63,14 @@ function Notify(title, options = {}) {
     }
 }
 
+Notify.serviceWorkerSupported = function() {
```

Comment:
hm... well, maybe a more accurate name would be `serviceWorkerNotificationsApiSupported`, but that'd be a mouthful hehe. Other ideas?